projects/systelab-components/src/lib/listbox/abstract-api-listbox.component.spec.ts

```diff
@@ -53,6 +53,7 @@ export class SystelabListboxComponent extends AbstractApiListBox<TestData> {
 		values.push(new TestData('1', 'Description 1'));
 		values.push(new TestData('2', 'Description 2'));
 		values.push(new TestData('3', 'Description 3'));
+		values.push(new TestData(4, 'Description 4'));
```

Comment:
Why do you have used a number instead of the string '4'?

Reply:
Because the error found with this type of list occurs when the id of the element is numeric, since a string (id of the node) is compared with a numeric type (id of the element), therefore the change made to solve it has been to compare the id of the element with the id of the data that is stored in the node instead of with the id of the node and for this reason a case in which the identifier is numeric has been added for testing.
![tempsnip](https://user-images.githubusercontent.com/48439749/150734746-9facb4ef-f899-4c5e-b313-1096588acc9d.png)